lms/djangoapps/progress/signals.py

```diff
@@ -26,6 +45,8 @@ def handle_cmc_post_save_signal(sender, instance, created, **kwargs):
     """
     content_id = unicode(instance.content_id)
     detached_categories = getattr(settings, 'PROGRESS_DETACHED_CATEGORIES', [])
+    if 'html' in content_id:
```

Comment:
Can you please add a comment explaining why you need this special case behavior?

(I assume it's because html blocks can be children of non-progress blocks like GPv2, and need to be ignored in that case)